pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -180,17 +180,22 @@ func (m *managerImpl) Start() error {
 	if err != nil {
 		return err
 	}
+	ticker := time.NewTicker(dbusReconnectPeriod)
+	defer ticker.Stop()
```

Comment:
does anything wong here?

Since below logic is running in another goroutine, so this func will return without any block, and `defer` will be executed very soon.

Reply:
Yes, I should put it in new goroutine